File: programmers/level1/2016년.py
def solution(a, b):
    day = -1
    for m in range(1, a):
        if m == 2: day += 29
        elif m in [2,4,6,9,11]: day += 30
        else: day += 31
    day += b
    day_name = ['FRI','SAT','SUN','MON','TUE','WED','THU']
    return day_name[day%7]

File: programmers/level1/2016년.java
class Solution {
    public String solution(int a, int b) {
        String[] days = {"FRI","SAT","SUN","MON","TUE","WED","THU"};
        int diffDays = 0;
        for(int i=1; i<a; i++) {
            if(i==2) diffDays += 29;
            else if(i<=7 && i%2==1) diffDays += 31;
            else if(i>7 && i%2==0) diffDays += 31;
            else diffDays += 30;
        }
        diffDays += b-1;
        diffDays = diffDays%7;
        return days[diffDays];
    }
}